Tests/Unit/NetworkServer/MessageProcessorSingleGatewayTest.cs

```diff
@@ -231,6 +219,7 @@ public void When_Faulty_MAC_Message_Is_Received_Processing_Abort_Without_Infinit
                 deviceRegistry,
                 FrameCounterUpdateStrategyProvider);
 
+            // This will be moved in #1086
```

Comment:
nit: to-do comments are easier to grep:

```suggestion
            // TODO This will be moved in #1086
```